src/index.html

```diff
@@ -8,6 +8,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="wrapper">
+
+
+      <div class="stars stars--0">
```

Comment:
```suggestion
    <div class="wrapper">
      <div class="stars stars--0">
```
Fix code style all over page